### ISSUE TITLE
Request hyperparameters for the Twitter dataset

### ISSUE THREAD

**Issue by cyang810** [AhaChang/DEFEND#1]
Thank you for the excellent work you have done.

I succeeded in replicating the results from the Reddit dataset, which are the same as your paper; 

Since the instructions for Twitter data set were missing and I noticed that the learning rate of Twitter data set was set to 0.005 in the paper, I adopted `python train.py --data Twitter --alpha 0.5 --gamma 1.5 --weight_corr 1e-10 --num_trial 10 --lr0 0.005` for training and the results are as follows:

`AUC: 0.6088±0.0081|PR: 0.0793±0.0017|SP: 0.0028±0.0019|EOO: 0.0125±0.0067|`

I guess the reason for the inconsistency between the results and the paper may be the existence of other hyperparameter changes. Could you please provide the results for me to reproduce the Twitter data set.

Looking forward to your reply Sincerely!!!

**Comment by AhaChang** [comment]
Hi, thanks for your interest.  
You can run the following code on Twitter:

```python
python train.py --data twitter --alpha 0.1 --gamma 2.5 --weight_corr 5e-15 --num_trial 10 --lr0 0.005
